packages/bpk-component-banner-alert/readme.md

```diff
@@ -34,53 +34,26 @@ export default () => (
 import React, { Component } from 'react';
 import BpkBannerAlert, { ALERT_TYPES } from 'bpk-component-banner-alert';
 
-class DismissableBpkBannerAlert extends Component {
-  constructor() {
-    super();
-
-    this.setDismissed = this.setDismissed.bind(this);
-
-    this.state = {
-      show: true,
-    };
-  }
-
-  setDismissed() {
-    this.setState({
-      show: false,
-    });
-  }
-
-  render() {
-    return (
-      <BpkBannerAlert
-        message="Successful alert with more information."
-        type={ALERT_TYPES.SUCCESS}
-        dismissable
-        onDismiss={this.setDismissed}
-        show={this.state.show}
-        dismissButtonLabel="Dismiss"
-      />
-    );
-  }
-}
-
 export default () => (
-  <DismissableBpkBannerAlert />
+  <BpkBannerAlert
+    message="Successful alert with dismiss option."
+    type={ALERT_TYPES.SUCCESS}
+    dismissable
+    dismissButtonLabel="Dismiss"
+  />
 );
 ```
 
 ## Props
 
-| Property          | PropType             | Required | Default Value |
-| ----------------- | -------------------- | -------- | ------------- |
-| type              | ALERT_TYPES (one of) | true     | -             |
-| message           | node                 | true     | -             |
-| ariaLive          | ARIA_LIVE (one of)   | false    | 'assertive'   |
-| children          | node                 | false    | null          |
-| dismissable       | bool                 | false    | false         |
-| dismissLabel      | string               | false    | null          |
```

Comment:
Just changed `dismissLabel` to `dismissButtonLabel` (the prop was wrong here), but I had to change the width of the whole table 😅

Reply:
Good catch!